hoodie-client/src/main/java/com/uber/hoodie/config/HoodieIndexConfig.java

```diff
@@ -49,6 +49,9 @@
   public final static String HBASE_ZKQUORUM_PROP = "hoodie.index.hbase.zkquorum";
   public final static String HBASE_ZKPORT_PROP = "hoodie.index.hbase.zkport";
   public final static String HBASE_TABLENAME_PROP = "hoodie.index.hbase.table";
+  public final static String HBASE_READ_BATCH_SIZE_PROP = "hoodie.index.hbase.read.batch.size";
```

Comment:
lets call these get and put batch sizes instead?

Reply:
Yeah, that makes more sense, duh